crates/jstzd/src/protocol/mod.rs

```diff
@@ -42,6 +44,8 @@ impl Protocol {
     fn hash(&self) -> &'static str {
         match self {
             Protocol::Alpha => "ProtoALphaALphaALphaALphaALphaALphaALphaALphaDdp3zK",
+            Protocol::ParisC => "PsParisCZo7KAh1Z1smVd9ZMZ1HHn5gkzbM94V3PLCpknFWhUAi",
+            Protocol::Quebec => "PsQubecQubecQubecQubecQubecQubecQubecQubecQubec",
```

Comment:
This is surely not the correct hash for quebec but it's fine for now since the actual protocol hash for Q is not yet determined anyway